frontend/tests/unit/models/public-schools-analysis-test.js

```diff
@@ -0,0 +1,172 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+
+module('Unit | Model | public schools analysis', function(hooks) {
+  setupTest(hooks);
+  setupMirage(hooks);
+
+  test('calculates currentMultiplier correctly, tests that currentMultiplier is now multipliers property in futureResidentialDev', async function(assert) {
+    let analysisMirage = server.create('public-schools-analysis');
+    let project = await this.owner.lookup('service:store').findRecord(
+      'project', analysisMirage.projectId, { include: 'public-schools-analysis' }
+    );
+    let analysis = await project.get('publicSchoolsAnalysis');
+
+    assert.equal(analysis.district, 17); // district is this.subDistrictFromDb[0].district
```

Comment:
district === currentMultiplier is just a random coincidence? If so, can you change the numbers so that is not the case? Makes reading the test confusing. Alternatively, create a variable called multiplier, set it to 17 and use that for the equality changes on line 17 and 19, but not this line making it explicit that the values are the same actual number but different theoretical variables